script/deploy.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+set -e
+set -o pipefail
+
+if [ -n "$(git status --porcelain)" ]; then
+  echo "The working directory is not clean. Please commit or stash your changes first."
+  exit 1
+fi
+
+# check if the current branch is `gh-pages`
+currentBranch=$(git rev-parse --abbrev-ref HEAD)
+if [[ ${currentBranch} == "gh-pages" ]]
+then
+  git push origin gh-pages
+else
+  # delete gh-pages branch and start a clean one
+  git branch -D gh-pages || true
+  git checkout -b gh-pages
+
+  # remove .gitignore
+  rm .gitignore
+
+  # build
+  npm run build
+
+  # commmit the new dist folder
+  git add dist
+  git commit -m ':rocket: deploy new version'
```

Comment:
Nice commit message! I might change it to be an argument because this message is what will show up at the bottom of `gh-board` when a new version is available (asking you to refresh your browser)